objects/CampaignImporterExporter.334ee3.json

```diff
@@ -0,0 +1,57 @@
+{
+  "AltLookAngle": {
+    "x": 0,
+    "y": 0,
+    "z": 0
+  },
+  "Autoraise": true,
+  "ColorDiffuse": {
+    "b": 1,
+    "g": 1,
+    "r": 1
+  },
+  "CustomImage": {
+    "CustomTile": {
+      "Stackable": false,
+      "Stretch": true,
+      "Thickness": 0.2,
+      "Type": 0
+    },
+    "ImageScalar": 1,
+    "ImageSecondaryURL": "",
+    "ImageURL": "http://cloud-3.steamusercontent.com/ugc/254843371583173230/BECDC34EB4D2C8C5F9F9933C97085F82A2F21AE3/",
+    "WidthScale": 0
+  },
+  "Description": "Saves the state of the table to enable loading the campaign into a new save and keep all current progress.\n\nThis tool will track which campaign you're playing, the entire contents of your campaign log, the contents of your chaos bag, update your health/sanity according to trauma, your ArkhamDB deck IDs, the number of investigators, the page of your campaign guide, and any options you have selected in the options panel.",
```

Comment:
Unless I'm missing something with the tagging, this is only going to work for official campaigns and not anything fanmade, correct?  Should make this very clear in the description, and possibly enhance various "not found" error messages with that.

Reply:
Well, this will work for a custom campaign provided they properly tag all their components (box, log, guide), and they have the campaign box on the table when the game is imported.  I've added that to the description